cli/tests/test_cmd_quickstart.py

```
@@ -238,6 +240,147 @@ def test_omitting_fail_mode_resolves_to_closed(self):
         raw = cfg["guardrail"].get("hook_fail_mode", "")
         self.assertEqual(_normalize_hook_fail_mode(raw), "closed")
 
+    def test_requested_gateway_start_failure_is_nonzero_in_human_and_json(self):
+        def invoke(args):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `invoke`

(ANN202)